server/handlers/auth.go

```diff
@@ -1,12 +1,17 @@
 package handlers
 
 import (
+	"context"
 	"net/http"
 	"strings"
 
 	"github.com/lesterfernandez/roommate-finder/server/token"
 )
 
+type ctxKey string
+
+var ContextKey ctxKey
```

Comment:
After setting this variable in `AuthenticateRoute`, it never changes. Could we turn this into a `const` and set its value here instead?